elements/lisk-cryptography/test/nacl/nacl.spec.ts

```diff
@@ -77,11 +76,13 @@ describe('nacl', () => {
 				});
 
 				it('should return an uint8array', () => {
-					return expect(randomBuffer).to.be.instanceOf(Uint8Array);
+					expect(Object.prototype.toString.call(randomBuffer)).toEqual(
```

Comment:
`.toBeInstanceOf`. same for below usage or Object prototype

Reply:
At this case it is not possible, due Jest has a custom implementation on the Buffer class, which cause that every child of Buffer like `Uint8Array` only return Buffer.